### ISSUE TITLE
pin an app to only run on a desktop

### ISSUE THREAD

**Issue by pcause** [mzomparelli/zVirtualDesktop#36]
What i'm really looking for is a way to get an app to always run on a specific desktop.  i'd hoped pin app/window would do this but it does the opposite (all desktops).  is there a way to add an "always open app X and all windows on desktop Y" to the app?

**Comment by mzomparelli** [comment]
@pcause I love this idea. This should be pretty easy to add. It might show up on the current desktop at first then be quickly and automatically moved to the desktop of your choosing. Would that be an acceptable implementation?

**Comment by mzomparelli** [comment]
Another thing we could do...picture this.

You want all Chrome windows to open on desktop 2 and you are currently on desktop 1. When you open Chrome, the Window will be moved and you will automatically follow it to desktop 2. You would be able to configure it to follow or not.

**Comment by mzomparelli** [comment]
Now that I think a little more, I guess you would always want to follow it otherwise you may not even know it opened.

**Comment by mzomparelli** [comment]
@pcause what would you name this effect? Pin is already used so we need a name for this.

**Comment by pcause** [comment]
thanks.  How about "app affinity" or "wired desktop"

**Comment by mzomparelli** [comment]
Those are options. Let's keeping thinking of a name. I was thinking of keeping it close to Pin and calling it Glue.

Glue the Chrome app to desktop 2

**Comment by Aran-Fey** [comment]
How about "Auto Move Windows"? I think that makes the difference to "pin"-ing very clear.

**Comment by mzomparelli** [comment]
Makes sense. Thanks for the feedback. I should get back to adding this feature soon.
